app/assets/src/components/views/compare/SamplesHeatmapView/SamplesHeatmapControls.jsx

```diff
@@ -414,7 +416,38 @@ export default class SamplesHeatmapControls extends React.Component {
     return filterTags;
   };
 
+  renderFilterStatsInfo = () => {
+    let { filteredTaxaCount, totalTaxaCount } = this.props;
+    return (
+      <span className={cs.reportInfoMsg}>
+        Showing {filteredTaxaCount} taxa of {totalTaxaCount} preselected taxa.
+        <ColumnHeaderTooltip
+          trigger={
+            <span>
+              <InfoSIcon className={cs.infoIcon} />
+            </span>
+          }
+          content="The data included in this heatmap 
+            was preselected based on the following conditions:"
+          list={[
+            "The top 1,000 unique taxa per sample",
+            "Only taxa with at least 5 reads",
+          ]}
+        />
+      </span>
+    );
+  };
+
   render() {
+    const { selectedOptions, loading, displayFilterStats } = this.props;
+    const { thresholdFilters, categories, subcategories } = selectedOptions;
+    // Only display the filter tag row if relevant filters are selected,
+    // otherwise an empty row will be rendered.
+    let displayFilterTags =
+      (thresholdFilters && thresholdFilters.length > 0) ||
```

Comment:
nit: `(thresholdFilters || []).length` would be shorter...

Reply:
Done!